app/services/handover_date_service.rb

```diff
@@ -1,24 +1,58 @@
 # frozen_string_literal: true
 
 class HandoverDateService
-  HandoverData = Struct.new :start_date, :handover_date, :reason
+  Responsibility = Struct.new(:description, :custody?, :case_owner) do
+    def to_s
+      description
+    end
+  end
+
+  RESPONSIBLE = Responsibility.new 'Responsible', true, 'Custody'
+  SUPPORTING = Responsibility.new 'Supporting', false, 'Community'
+  UNKNOWN = Responsibility.new 'Unknown', false, 'Unknown'
+
+  # Actual date Mon 19th Oct 2020
+  PRESCOED_CUTOFF_DATE = Date.new(2020, 10, 19).freeze
+
+  HandoverData = Struct.new :responsibility, :start_date, :handover_date, :reason
 
   # if COM responsible, then handover dates all empty
-  NO_HANDOVER_DATE = HandoverData.new nil, nil, 'COM Responsibility'
+  NO_HANDOVER_DATE = HandoverData.new SUPPORTING, nil, nil, 'COM Responsibility'
+
+  def self.calculate_pom_responsibility(offender)
+    handover(offender).responsibility
+  end
+
+  def self.handover(raw_offender)
+    offender = OffenderWrapper.new(raw_offender)
 
-  def self.handover(offender)
     if offender.recalled?
-      HandoverData.new nil, nil, 'Recall case - no handover date calculation'
-    elsif offender.nps_case? && offender.indeterminate_sentence? && offender.tariff_date.nil?
-      HandoverData.new nil, nil, 'No earliest release date'
-    elsif ResponsibilityService.recent_prescoed_nps_case?(offender) && offender.indeterminate_sentence?
-      HandoverData.new offender.prison_arrival_date, prescoed_handover_date(offender), 'Prescoed'
+      HandoverData.new SUPPORTING, nil, nil, 'Recall case'
+    elsif offender.immigration_case?
+      HandoverData.new SUPPORTING, nil, nil, 'Immigration Case'
+    elsif offender.nps_case? && offender.indeterminate_sentence? && (offender.tariff_date.nil? || offender.tariff_date < Time.zone.today)
+      HandoverData.new RESPONSIBLE, nil, nil, 'No earliest release date'
```

Comment:
@starswan As discussed on Slack earlier today – there is a discrepancy between the Handover Date Service and the Responsibility Service when calculating on **an indeterminate offender with a Tariff Date in the past**.

**Handover Date Service** would still give handover dates, even though they're in the past:

https://github.com/ministryofjustice/offender-management-allocation-manager/blob/0111140c8e55c79818a79a0b6e6d81b30f143ee2/app/services/handover_date_service.rb#L16-L18

And the **Responsibility Service** would say the POM is responsible:

https://github.com/ministryofjustice/offender-management-allocation-manager/blob/0111140c8e55c79818a79a0b6e6d81b30f143ee2/app/services/responsibility_service.rb#L30-L33

The change made in this PR would change the behaviour:
- Handover dates become `nil` (where they were previously dates in the past)
- Responsibility remains unchanged as `'Responsible'`

I'm making this comment to flag that _I think_ this refactor changes the behaviour for these offenders. Is that the expected behaviour?

Reply:
One for Laura I think. If the tariff_date is in the past but the offender is still in prison, IMHO it's the same as not having a tariff date - they have failed their parole hearing, have a PED or a PRD (obviously we can't get PRD). I don't think I changed any tests substantially which would indicate that this behaviour has changed? [Not that that proves anything, just that we might have an uncovered case - bit annoying if so]

Reply:
Looks like it used to be covered by a test:

https://github.com/ministryofjustice/offender-management-allocation-manager/blob/bac5eba830917919ca61a81ec7db0ccf22abad32/spec/services/responsibility_service_spec.rb#L19-L32

But this has since been refactored / removed from the codebase, and I don't think we explicitly cover this scenario any more.